packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -257,7 +270,21 @@ function buildMockMessenger({
     });
   }
 
+  const mockSubscribe = jest.fn();
+  mockSubscribe.mockImplementation((_type, handler) => {
+    setTimeout(() => {
+      handler({}, [
+        {
+          op: 'add',
+          path: ['networkConfigurations', 'foo', 'bar'],
```

Comment:
```suggestion
          path: ['networkConfigurations', 'foo'],
```

makes this a bit more accurate